src/main/java/seedu/easylog/commands/orderscommands/OrdersAddCommand.java

```diff
@@ -17,8 +17,8 @@
 public class OrdersAddCommand extends OrdersCommand {
 
     public void execute(String customerName, ItemManager itemManager, OrderManager orderManager)
-            throws EmptyNameException, EmptyItemListException, RepeatedOrderException {
-        if (customerName.equals("")) {
+            throws EmptyNameException, EmptyItemListException {
+        if (customerName.trim().equals("")) {
             throw new EmptyNameException();
         }
```

Comment:
think there is no need to trim the customer name here because all inputs should be trimmed at the start alrd, can just go back to normal and just check for .equals(""). I think what needs to be trimmed is when asking for the item index and item quantity to be inputted into the order thats need to be trimmed